src/core/config/npo/snapshots/tailcall__core__config__npo__tracker__tests__multiple_keys.snap

```diff
@@ -2,5 +2,5 @@
 source: src/core/config/npo/tracker.rs
 expression: actual
 ---
-query { f1 { f2 } }
 query { f1 { f3 } }
```

Comment:
just the order change, I'm not sure why it's inconsistent for different test cases